tests/Tests.fs

```diff
@@ -874,7 +874,8 @@ let private serializationTests =
                     Data = None }
                testThereAndBackAgain item
         ]
-      Shotgun.tests ]
+      Shotgun.tests
+      StartWithSetup.tests ]
 
 [<Tests>]
 let tests = testList "LSP" [ serializationTests; Utils.tests ]
```

Comment:
I just notice with Utils.Tests being here, maybe StartWithSetup.tests should be too, rather than at the end of the serializationTests. Dunno what maintainers want.
`let tests = testList "LSP" [ serializationTests; Utils.tests; StartWithSetup.tests ]`